cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -116,6 +116,35 @@
   }
 }
 
+.m-apr-rating {
+  &--less .cf-icon-svg {
+    color: var(--green);
+  }
+
+  &--average .cf-icon-svg {
+    color: var(--gold);
+  }
+
+  &--more .cf-icon-svg {
+    color: var(--red);
+  }
+}
+
+.heading-4-size-only() {
```

Comment:
Prefix this with a `u-`, so that it is `.u-heading-4-size-only()` … that way it's clearer that this should be used as a mixin in utility and not directly in the markup.